tck/faces-tck/src/com/sun/ts/tests/jsf/api/jakarta_faces/view/statemanagementstrategy/URLClient.java

```diff
@@ -57,20 +57,7 @@ public Status run(String args[], PrintWriter out, PrintWriter err) {
    * @since 2.2
    */
   public void stateMgmtStrategyNonNullTest() throws Fault {
-    TEST_PROPS.setProperty(APITEST, "stateMgmtStratNullForJSPTest");
-    invoke();
-  }
-
-  /**
-   * @testName: stateMgmtStratNullForJSPTest
-   * @assertion_ids: JSF:JAVADOC:2243
-   * @test_Strategy: Validate that StateManagementStrategy is null for JSP
-   *                 views.
-   * 
-   * @since 2.2
-   */
-  public void stateMgmtStratNullForJSPTest() throws Fault {
-    TEST_PROPS.setProperty(APITEST, "stateMgmtStratNullForJSPTest");
+    TEST_PROPS.setProperty(APITEST, "stateMgmtStrategyNonNullTest");
```

Comment:
I noticed this tested `stateMgmtStratNullForJSPTest` when the method name was actually `stateMgmtStrategyNonNullTest`. I corrected it.